src/components/ContactCard/ContactForm/ContactFormField.jsx

```diff
@@ -36,16 +38,14 @@ const ContactFormField = ({ name, icon, isArray, renderInput }) => {
         {renderInput(`${nameWithIndex}.${name}`, fieldID)}
 
         {hasValue && (
-          <Button
-            type="button"
-            theme="secondary"
-            label="delete"
-            iconOnly
-            round
-            icon={'cross-small'}
+          <IconButton
             onClick={() => removeField(fields, index)}
-            className="u-pos-absolute u-top-0 u-right-0 u-mt-1"
-          />
+            className="contact-form-field__delete-icon u-top-0 u-right-0 u-mt-1"
```

Comment:
why did you create a new class instead of using `u-pos-absolute` as before?

Reply:
`u-pos-absolute` can't override `IconButton` because it's CSS selector weight is lower than css class of the button that define `position: relative`. `u-pos-absolute` miss an `!important` but it's hard to determine all the regression by adding it directly to Cozy UI.

Reply:
OK see https://github.com/cozy/cozy-ui/issues/2284